README.rst

```diff
@@ -126,15 +149,13 @@ Summary:
 Continuous Integration Status
 =============================
 
-Travis builds (Linux & OS X):
-
-.. image:: https://travis-ci.org/Cantera/cantera.svg?branch=master
-    :target: https://travis-ci.org/Cantera/cantera
-
-Appveyor builds (Windows):
+==============  ============  ===================
+Platform        Site          Status
+==============  ============  ===================
+Linux & OS X    Travis CI     |travisci|
+Windows x64     Appveyor      |appveyor|
+==============  ============  ===================
```

Comment:
I like this -- I hadn't realized that using the `|label|` substitutions would let you put images into a table.